addons/l10n_mn/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    "name" : "Mongolia - Accounting",
+    "version" : "1.0",
+    'category': 'Localization',
+    "description": """
+This is the module to manage the accounting chart for Mongolia.
+===============================================================
+
+    * the Mongolia Official Chart of Accounts,
+    * the Tax Code Chart for Mongolia
+    * the main taxes used in Mongolia
```

Comment:
Can we add a reference to their contribution at least?  Other partners are in copyright and everything? 

Financial requirement contributor: Baskhuu Lodoikhuu. BumanIT LLC

Reply:
Will do it myself